### PR TITLE
docs: Add Data Driven Insights Page from Blog

### PR DESCRIPTION
**Pull request by jfermi**
Remove "Pro Feature" label from pages.
Redirect "legacy warning" to a file.

This PR...

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/articles/data-driven-insights.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+![Execution Metrics](../img/execution-metrics.png)
+
+**Performance Analysis:** Assess the execution duration of tests to identify which tests take the longest and may need optimization to reduce application delivery times and infrastructure costs.
+
+![Performance Analysis](../img/performance-analysis.png)
+
+**Test Categorization:** Filter insights by test types (load, API, end-to-end) or by testing tools, enabling teams to pinpoint specific areas for improvement.
+
+## The Value of Test Insights
```
Comment (olensmar) [inline docs/articles/data-driven-insights.md:36]: move this to the top - second section?

File: docs/articles/data-driven-insights.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+By integrating Test Insights into your testing workflow, your team can leverage several benefits:
+
+- **Enhanced Visibility:** Gain a clear overview of testing activities and outcomes, helping you manage and prioritize testing resources more effectively.
+- **Data-Driven Decisions:** Make informed decisions about where to allocate efforts for improving test efficiency and reliability.
+- **Improved Test Efficiency:** Identify and address inefficiencies in your testing process, leading to faster development cycles and higher quality software releases.
+
+## Getting Started with Test Insights
```
Comment (olensmar) [inline docs/articles/data-driven-insights.md:44]: let's remove this and everything below - this is documentation, not marketing material :D

File: docs/articles/data-driven-insights.md (hunk 1 above)
Comment (jfermi) [reply docs/articles/data-driven-insights.md:44]: Done.

File: docs/articles/data-driven-insights.md (hunk 2 above)
Comment (jfermi) [reply docs/articles/data-driven-insights.md]: Done.

File: docs/articles/data-driven-insights.md (hunk 1 above)
Comment (olensmar) [reply docs/articles/data-driven-insights.md:44]: it looks like this section is now at both the top and bottom!? https://testkube-docs-git-julianne-docscreate-dat-9e4c15-testkube-cloud.vercel.app/articles/data-driven-insights#the-value-of-test-insights-1

Comment (jfermi) [reply docs/articles/data-driven-insights.md]: Fixed.

**Review by olensmar — APPROVED** [review]

**Pull request merged**